Source/React.cpp

```diff
@@ -425,6 +425,30 @@ PeleC::react_state(
         } else {
           amrex::Abort("chem_integrator must be equal to 1, 2, or 3");
         }
+
+        // update heat release
+        amrex::ParallelFor(
+          bx, [=] AMREX_GPU_DEVICE(int i, int j, int k) noexcept {
+            I_R(i, j, k, NUM_SPECIES + 1) = 0.0;
+            auto eos = pele::physics::PhysicsType::eos();
+
+            amrex::Real hi[NUM_SPECIES] = {0.0};
+
+#ifndef PELEC_USE_SRK
```

Comment:
sure, it will ignore Y as well?

Reply:
RTY2Hi doesnt exist when SRK is disabled

Reply:
Yeah it should be fine... if it existed. But we don't have a dummy Fuego `RTY2Hi` that mirrors `TY2Hi`. So the right fix would be to add in PelePhysics. But we have other `USE_SRK` defs in the code that will have to be lifted some time and we can address that then.

Reply:
Yeah, if there were an RTY2Hi for Fuego it would just wrap around T2Hi which in turn just wraps CKHMS(&T, Hi). It looks like we didn't implement the RTY2Hi wrapper in the Fuego EOS because it only shows up in PeleC for the nonideal transport calculation.